jobs/Llama2/original_use/job_original_use_7b_hf.sh

```diff
@@ -0,0 +1,30 @@
+#!/bin/bash
+#SBATCH --job-name=llama2-orig-use-7b-hf
+#SBATCH --time 10-00:00
+#SBATCH --nodes=1
+#SBATCH --gpus-per-node=v100l:1
+#SBATCH --mem=128G
+#SBATCH --ntasks-per-node=4
+#SBATCH --cpus-per-task=4
+#SBATCH --account=def-whkchun
+#SBATCH --output=%x.o%j
```

Comment:
These parameter values are experimental. Before I got the job completed successfully, I had no idea how long it will need. 10 days was plenty so I used it as an initial value. It turned out 1 hour should be enough. Regarding the GPU node, I read [this documentation](https://docs.alliancecan.ca/wiki/Using_GPUs_with_Slurm). I will add it in the "References" section in the Llama2FineTuning.md since all job scripts require it.